src/main/java/com/tenx/ms/retail/order/domain/OrderEntity.java

```diff
@@ -0,0 +1,129 @@
+package com.tenx.ms.retail.order.domain;
+
+import com.tenx.ms.commons.validation.constraints.Email;
+import com.tenx.ms.commons.validation.constraints.PhoneNumber;
+import com.tenx.ms.retail.order.domain.enums.OrderStatusEnum;
+import com.tenx.ms.retail.store.domain.StoreEntity;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+import java.sql.Timestamp;
+import java.util.List;
+
+@Entity
+@Table(name = "order_complete")
+public class OrderEntity {
+    @Id
+    @GeneratedValue
```

Comment:
You should specify the generation strategy. Just for readability and good practice.